packages/suite/package.json

```diff
@@ -82,7 +82,6 @@
         "dropbox": "^10.34.0",
         "file-saver": "^2.0.5",
         "framer-motion": "^11.0.3",
-        "history": "^4.10.1",
```

Comment:
You'll also want to remove the history dependency from your package.json. The history library is a direct dependency of v6 (not a peer dep), so you won't ever import or use it directly. Instead, you'll use the useNavigate() hook for all navigation (see below).